### DISCUSSION TITLE
Policy for VolumeClaims

### DISCUSSION THREAD

**Discussion by RajeevPatel7** [kyverno/policies#256] [General]
I am using NFS provisioner for PVC provisioning. I have storage class defined which accepts PVC.annotations.nfs.io/storage-path for mounting the volume. I would like to validate the "nfs.io/storage-path" annotation that it should not be empty to avoid mounting on root folder

**Comment by chipzoller** [comment]
Please use the appropriate issue form and request a policy.

**Reply by chipzoller** [reply]
@RajeevPatel7 will you create a formal policy request, please?

https://github.com/kyverno/policies/issues/new?assignees=&labels=sample&template=sample_policy.yml&title=%5BSample%5D+

**Comment by chipzoller** [comment]
This has been captured in https://github.com/kyverno/policies/issues/313 and will be closed shortly.
